.github/workflows/test.yml

```diff
@@ -30,4 +30,3 @@ jobs:
       - name: Run tests
         timeout-minutes: 5
         run: pytest tests/
-
```

Comment:
I won't reject because of this, but don't change workflow files for the next time
Overall, good job!

Reply:
Yes, I have been changed this file, but then I returned all changes